dhcp/unikernel.ml

```diff
@@ -0,0 +1,85 @@
+open V1_LWT
+open Lwt
+
+(* IP Configuration, all you need besides dhcpd.conf. *)
+let ipaddr = Ipaddr.V4.of_string_exn "192.168.1.5"
```

Comment:
Does that mean we should wait until functoria is properly released before adding this?

> On 12 Nov 2015, at 17:12, Thomas Gazagnaire <notifications@github.com> wrote:
> 
> In dhcp/unikernel.ml:
> 
> > @@ -0,0 +1,85 @@
> > +open V1_LWT
> > +open Lwt
> > +
> > +(* IP Configuration, all you need besides dhcpd.conf. *)
> > +let ipaddr = Ipaddr.V4.of_string_exn "192.168.1.5"
> 
> it'd be very easy to do with the new functoria, see https://github.com/Drup/mirage-skeleton/blob/functoria/hello/config.ml#L3-L5
> 
> —
> Reply to this email directly or view it on GitHub.
>